content/en/security/logs.md

```diff
@@ -17,7 +17,9 @@ The Log Management product supports multiple [environments and formats][2], allo
 
 ## Information Security
 
-The Datadog Agent submits logs to Datadog over a TLS-encrypted TCP connection, requiring an outbound communication over port `10516`. The Datadog Agent can also be configured to send logs over secured HTTPS requests, requiring an outbound communication over port `443`. Datadog uses symmetric encryption at rest (AES-256) for indexed logs. Indexed logs are deleted from the Datadog platform once their retention period, as defined by the customer, expires.
+The Datadog Agent submits logs to Datadog via HTTPS when possible. Logs are sent over port `443` with compression. Datadog uses symmetric encryption at rest (AES-256) for indexed logs. Indexed logs are deleted from the Datadog platform once their retention period, as defined by the customer, expires.
```

Comment:
I would go for a shorter version which don't specify the behavior of old version. 
"The Datadog Agent submits logs to data through HTTPS or through TLS-encrypted TCP connection on port `10516`, requiring outbound communication (see [Agent Transport for logs][8]).

Reply:
changed the wording to match the suggestion of this comment